src/components/standard/article.tsx

```diff
@@ -78,6 +78,25 @@ interface Props {
     children: ReactNode[];
 }
 
+const lineStyles = css`
+    ${from.wide} {
+        width: ${wideContentWidth}px;
+        margin-left: auto;
+        margin-right: auto;
+    }
+    div {
```

Comment:
These styles appear in three different places, would it be a good idea defining it once and importing where they're used?

Reply:
I have now placed all in one class in the styles.tsx
Note: I have kept the style customised for liveblog articles as they are different

Reply:
Nice, that'll be easier to delete when source adds dark mode support